qiita_db/metadata_template.py

```diff
@@ -760,6 +760,13 @@ def _check_special_columns(cls, md_template, obj):
             The obj to which the metadata template belongs to. Study in case
             of SampleTemplate and RawData in case of PrepTemplate
         """
+        # Check disallowed col names
+        disallowed = {'study_id', 'processed_data_id'}
```

Comment:
Note also, though, that Jose's idea violates the prime directive of qiita_db objects never importing from any other qiita_db object file. We've kept that going until now, so we should NOT break that in my opinion.